clang/docs/ReleaseNotes.rst

```diff
@@ -145,6 +145,7 @@ Improvements to Clang's diagnostics
   prints.
 
 - Clang now diagnoses member template declarations with multiple declarators.
+- Clang now diagnoses friend declarations with an ``enum`` elaborated-type-specifier outside of C++98.
```

Comment:
Do you mean, "in language modes after C++98"?

Reply:
Yes :)

Reply:
Ok, cool, I was a touch confused.  Can you change the wording of the release note?

Reply:
Yup, will do